package.json

```diff
@@ -1,62 +1,59 @@
-    "start": "skyux serve --hmr",
-    "test": "skyux test --coverage library --logFormat none",
-    "test:visual": "skyux e2e --logFormat none"
-  },
-  "keywords": [],
-  "author": "Blackbaud",
-  "license": "ISC",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/blackbaud/skyux-sdk-testing.git"
-  },
-  "bugs": {
-    "url": "https://github.com/blackbaud/skyux-sdk-testing/issues"
-  },
-  "homepage": "https://github.com/blackbaud/skyux-sdk-testing#readme",
-  "peerDependencies": {
-    "@angular/core": "^9.1.9",
-    "@skyux/config": "^4.0.0"
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test",
+    "skyux:generate-lib-resources-module": "ng generate @skyux/i18n:lib-resources-module",
+    "lint": "ng lint",
+    "skyux:generate-documentation": "ng generate @skyux-sdk/documentation-schematics:documentation",
+    "postbuild": "npm run skyux:generate-documentation"
   },
+  "private": true,
   "dependencies": {
-    "axe-core": "3.5.6"
+    "@angular/animations": "12.2.7",
+    "@angular/common": "12.2.7",
+    "@angular/compiler": "12.2.7",
+    "@angular/core": "12.2.7",
+    "@angular/forms": "12.2.7",
+    "@angular/platform-browser": "12.2.7",
+    "@angular/platform-browser-dynamic": "12.2.7",
+    "@angular/router": "12.2.7",
+    "@blackbaud/auth-client": "2.50.0",
+    "@skyux/i18n": "5.0.0",
+    "axe-core": "3.5.6",
+    "rxjs": "6.6.7",
+    "tslib": "2.3.1",
+    "zone.js": "0.11.4"
   },
   "devDependencies": {
-    "@angular/animations": "9.1.13",
-    "@angular/common": "9.1.13",
-    "@angular/compiler": "9.1.13",
-    "@angular/compiler-cli": "9.1.13",
-    "@angular/core": "9.1.13",
-    "@angular/forms": "9.1.13",
-    "@angular/platform-browser": "9.1.13",
-    "@angular/platform-browser-dynamic": "9.1.13",
-    "@angular/router": "9.1.13",
-    "@blackbaud/auth-client": "2.48.0",
-    "@skyux-sdk/builder": "4.9.0",
-    "@skyux-sdk/builder-plugin-skyux": "4.2.0",
-    "@skyux-sdk/e2e": "4.0.2",
-    "@skyux/assets": "4.0.1",
-    "@skyux/config": "4.4.0",
-    "@skyux/core": "4.6.0",
-    "@skyux/http": "4.2.0",
-    "@skyux/i18n": "4.2.1",
-    "@skyux/indicators": "4.10.0",
-    "@skyux/modals": "4.5.4",
-    "@skyux/omnibar-interop": "4.0.1",
-    "@skyux/router": "4.2.2",
-    "@skyux/theme": "4.17.0",
-    "codelyzer": "5.2.2",
-    "rxjs": "6.6.7",
+    "@angular-devkit/build-angular": "~12.2.7",
+    "@angular-eslint/builder": "12.4.1",
+    "@angular-eslint/eslint-plugin": "12.4.1",
+    "@angular-eslint/eslint-plugin-template": "12.4.1",
+    "@angular-eslint/schematics": "12.4.1",
+    "@angular-eslint/template-parser": "12.4.1",
+    "@angular/cli": "~12.2.7",
+    "@angular/compiler-cli": "~12.2.0",
+    "@skyux-sdk/documentation-schematics": "^5.0.0-beta.4",
+    "@skyux/assets": "^5.0.0-beta.2",
+    "@skyux/config": "^5.0.0-beta.4",
+    "@skyux/packages": "^5.0.0-beta.132",
```

Comment:
Range characters should be removed from all `@skyux/*`, `@skyux-sdk/*`, `@angular/*`, and `@angular-devkit/*`.

Also, `@skyux/packages` should be listed in `dependencies`.